File: Python/Python-Chapter-One/One_10.py
# Assume a runner runs 14 km in 45 minutes and 30 seconds
# Write a program that displays the average speed in miles per
# hour. (Note that one mile is 1.6 km)
distance = 14 / 1.6
time = (45 + (30.0 / 60.0)) / 60.0
speed = distance / time
print("Speed in mph = ", speed)

File: Java/Chapter-One/One_10.java
/**
 * 1.10 
 * Assume a runner runs 14 kilometers in 45 minutes and 30 seconds. Write a
 * program that displays the average speed in miles per hour. (Note that 1 mile
 * is 1.6 kilometers)
 * 
 * @author Jack Donofrio
 * @date 1/19/20
 *
 */
public class One_10
{
  public static void main(String[] args)
  {
    double distance = 14; // kilometers
    double time = 45.5; // minutes
    // Convert kilometers to miles and minutes to hours.
    // Divide those values to get miles per hour
    double mph = (distance / 1.6) / (time / 60);
    System.out.print("14 kilometers in 45 minutes and 30 seconds is equal to "
      + mph + " miles per hour");
  }

}
